internal/pkg/runtime/engines/singularity/prepare_linux.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sylabs/singularity/internal/pkg/buildcfg"
-	"github.com/sylabs/singularity/internal/pkg/fakeroot"
+	fakerootUtil "github.com/sylabs/singularity/internal/pkg/fakeroot"
```

Comment:
I thought I remember go[fmt,lint,vet] said package name should not have caps? Does that apply here?

Reply:
Ah yeah I forgot this rule, will fix that. Thanks !